watertap/unit_models/MD/membrane_distillation_0D.py

```diff
@@ -0,0 +1,567 @@
+            description="Pressure change term construction flag",
+            doc="""
+        Indicates what type of pressure change calculation will be made. To use any of the
+        ``pressure_change_type`` options to account for pressure drop, the configuration keyword
+        ``has_pressure_change`` must also be set to ``True``. Also, if a value is specified for pressure
+        change, it should be negative to represent pressure drop.
+
+        **default** - ``PressureChangeType.fixed_per_stage`` 
+
+        .. csv-table::
+            :header: "Configuration Options", "Description"
+        
+            "``PressureChangeType.fixed_per_stage``", "Specify an estimated value for pressure drop across the membrane feed channel"
+            "``PressureChangeType.fixed_per_unit_length``", "Specify an estimated value for pressure drop per unit length across the membrane feed channel"
+            "``PressureChangeType.calculated``", "Allow model to perform calculation of pressure drop across the membrane feed channel"
+        """,
+        ),
+    )
+
+    _CONFIG_Template.declare(
+        "friction_factor",
+        ConfigValue(
+            default=FrictionFactor.flat_sheet,
+            domain=In(FrictionFactor),
+            description="Darcy friction factor correlation",
+            doc="""
+            Options to account for friction factor correlations.
+
+            **default** - ``FrictionFactor.flat_sheet`` 
+
+        .. csv-table::
+            :header: "Configuration Options", "Description"
+
+            "``FrictionFactor.flat_sheet``", "Friction factor correlation for flat-sheet membrane modules"
+            "``FrictionFactor.spiral_wound``", "Friction factor correlation for spiral-wound membranes"
+        """,
+        ),
+    )
+
+    _CONFIG_Template.declare(
+        "property_package_vapor",
+        ConfigValue(
+            default=useDefault,
+            domain=is_physical_parameter_block,
+            description="Property package to use for control volume",
+            doc="""Property parameter object used to define property calculations,
+        **default** - useDefault.
+        **Valid values:** {
+        **useDefault** - use default package from parent model or flowsheet,
+        **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    _CONFIG_Template.declare(
+        "property_package_args_vapor",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property packages",
+            doc="""A ConfigBlock with arguments to be passed to a property block(s)
+        and used when constructing these,
+        **default** - None.
+        **Valid values:** {
+        see property package for documentation.}""",
+        ),
+    )
+
+    CONFIG.declare("hot_ch", _CONFIG_Template(doc="hot channel config arguments"))
+    CONFIG.declare("cold_ch", _CONFIG_Template(doc="cold channel config arguments"))
```

Comment:
Minor preference here, but I think it'd be better to fully spell out `hot_channel` and `cold_channel`.

Reply:
We can do this in future PRs as it requires changing many lines of code.